prereise/gather/hiflddata/data_trans.py

```diff
@@ -0,0 +1,645 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import csv
+import json
+import os.path
+import zipfile
+from collections import defaultdict
+
+import networkx as nx
+import pandas as pd
+from haversine import Unit, haversine
+
+Max_Value = 3000
+Min_Value = 0
+coord_precision = ".9f"
+default_base_kv = -999999.0
```

Comment:
We should strive to avoid the use of fake data which might accidentally get used in calculations.

Reply:
This has been addressed in the latest version. We no longer apply such fake setup with -999999.0